DEVELOPMENT.md

```diff
@@ -0,0 +1,78 @@
+# Developement
+
+## Feature Release Flow
+
+### Branching Strategy
+
+- The `master` branch is intended to only contain features for the immediately upcoming release, whether it is a Height Coordinated Upgrade or a Spork
+- During development, all features should live on a feature branch.
+- For small features, this will be simple working branch, i.e. `kan/1-fix-known-issue`
```

Comment:
```suggestion
- For small features, this will be simple working branch. These branches have the naming scheme `<contributor>/<issue #>-<short description>, for example `kan/123-fix-known-issue`
```